tests/cli/test_main_run.py

```diff
@@ -118,3 +118,77 @@ def test_multiline_run_command(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixtur
         )
         assert stdout
         assert not stderr
+
+
+def test_run_deactivates_environment(
```

Comment:
The test cases are so significantly different that it might be easier to split the test function up into two functions, one for Windows, and one for non-Windows.

OR to use `@pytest.mark.parametrize` and refactor it to check for similar values.